package.json

```diff
@@ -55,6 +54,7 @@
     "yaml": "^1.10.2"
   },
   "devDependencies": {
+    "eslint-plugin-mocha": "^9.0.0",
```

Comment:
There is a newer major version of this package, but it drops support for Node.js v12, which [is not EOLed yet](https://nodejs.org/en/about/releases/). Once v12 is no longer in LTS, we can remove it from our testing matrix and bump up this dependency.